SocialFilter/sections/specific.txt

```diff
@@ -44,6 +44,7 @@ duzcedamla.com,muglagazetesi.com.tr,apron24.com,bilecikses.com,enzahome.com.tr,d
 !
 ehow.com##.component-pinterest
 ehow.com##.social-share-container
+speed-down.org#?#table > tbody > tr:has(> td > div.share)
```

Comment:
and this

Reply:
<details>
<summary>Screenshot</summary>
<img src="https://user-images.githubusercontent.com/18536328/88312549-eef37d80-cd1a-11ea-898c-64ba9c23c82e.png" />
</details>